rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -275,6 +275,67 @@
           "templateOptions.required": "model.roles.some(v => v === 'patron')"
         }
       }
+    },
+    "subscriptions": {
+      "title": "Patron subscriptions",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "title": "Subscription",
+        "type": "object",
+        "required": [
+          "start_date",
+          "end_date",
+          "patron_type",
+          "patron_transaction"
+        ],
+        "properties": {
+          "start_date": {
+            "title": "Subscription start date",
+            "description": "The subscription start date (date included)",
+            "type": "string",
+            "format": "date",
+            "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$"
```

Comment:
This pattern is not 100% correct (2222-02-31!) 
And do we need a pattern if we have `"format": "date"

Reply:
You are totally right, this pattern is used to validate the input data with formly editor. Using it, the user/librarian must at least used a correct date string format (YYYY-MM-DD) and not (DD-MM-YYYY). The `"format": "date"` is used to validate the the date string is correct using JSON schema validation.
I used the already used pattern from  "buget" and "acq_order" resources.